### PR TITLE
[#168660241] Fix the action of the service detail buttons

### PR DESCRIPTION
**Pull request by matgentili**
Now with this pr:
- When user press a button the text is copied and receives feedback
- Changed on IOS phone call management

### CODE REVIEW

**Comment by digitalcitizenship** [comment]
## Affected stories

  * 🐞 [#168660241](https://www.pivotaltracker.com/story/show/168660241): Verificare il corretto funzionamento dei bottoni nel dettaglio del servizio [IOS]

<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by Undermaken** [comment]
Is it still in draft?

**Comment by matgentili** [comment]
@Undermaken Yes, it is in draft. I have a problem with the screen background when I click on the call button.

**Comment by matgentili** [comment]
AFTER ANDROID
after andorid
AFTER IOS
iphone after

File: locales/it/index.yml [hunk 1]
```diff
@@ -1,5 +1,7 @@
 ---
 global:
+  alert:
+    call: "Vuoi chiamare #?"
```
Comment (Undermaken) [inline locales/it/index.yml]: There is a formal way to define variable inside a yml value
https://github.com/teamdigitale/io-app/blob/7c3d5e999070641893eafbb121e396aa3cbe74e5/locales/en/index.yml#L591

Btw could you add another line to report the number?
ex: 
```
Do you want to call SERVICE_NAME?
+39 1234 567
```

File: locales/en/index.yml [hunk 2]
```diff
@@ -1,5 +1,7 @@
 ---
 global:
+  alert:
+    call: "Do you want call #?"
```
Comment (Undermaken) [inline locales/en/index.yml]:
```suggestion
    call: "Do you want to call #?"
```

File: locales/it/index.yml (hunk 1 above)
Comment (matgentili) [reply locales/it/index.yml]: No longer necessary

File: locales/en/index.yml (hunk 2 above)
Comment (matgentili) [reply locales/en/index.yml]: No longer necessary

**Comment by matgentili** [comment]
@Undermaken I have removed the changes related to the background problem. You can review it, thanks :)

File: ts/screens/services/ServiceDetailsScreen.tsx [hunk 3]
```diff
@@ -447,7 +457,7 @@ class ServiceDetailsScreen extends React.Component<Props, State> {
               )}
             {phone &&
```
Comment (Undermaken) [inline ts/screens/services/ServiceDetailsScreen.tsx]: We could restore the previous version that we know it works and remove the method **openPhoneDialer**

```typescrip
{phone &&
              renderInformationRow(I18n.t("services.contactPhone"), phone, () =>
                Linking.openURL(`tel:${phone}`).then(() => 0, () => 0)
              )}
```

**Pull request merged**